src/shared/vars.js

```diff
@@ -1,4 +1,6 @@
-const VERSION = '__skate_0_14_0';
```

Comment:
It's probably used much less now since `skate` is no longer on the window, you'd have to figure out the version inside a closure.